versions.md

```diff
@@ -24,6 +24,10 @@ Forthcoming
   * TimeConversion
   * DBQueue
```

Comment:
To this list of no longer available public API we should add TypeLookup Service too

Reply:
Done in 4041064a0